neuralpp/symbolic/sympy_expression.py

```diff
@@ -57,8 +59,8 @@ def infer_sympy_object_type(sympy_object: sympy.Basic, type_dict: Dict[sympy.Bas
     (operator.add, sympy.Add),
     (operator.mul, sympy.Mul),
     (operator.pow, sympy.Pow),
-    (operator.sub, sympy.Lambda((abc.x, abc.y), abc.x-abc.y)),  # "lambda x: (-1)*x"
-    (operator.neg, sympy.Lambda((abc.x,), -abc.x)),  # "lambda x: (-1)*x"
+    (operator.sub, sympy_Sub),
+    (operator.neg, sympy_Neg),  # "lambda x: (-1)*x"
```

Comment:
Perhaps we should move the comments up to line 44 (where you define `sympy_Neg`?)